### ISSUE TITLE
Example how to implement with new API (https://github.com/react-navigation/react-navigation-redux-helpers/tree/2.0)

### ISSUE THREAD

**Issue by mrikirill** [react-navigation/react-navigation-redux-helpers#39]
Hi 👋 

Is there any example about how to implement react-navigation with new API [](https://github.com/react-navigation/react-navigation-redux-helpers/tree/2.0) ?
React-navigation docs provides the instruction only for the previous version [](https://reactnavigation.org/docs/en/redux-integration.html)

**Comment by Ashoat** [comment]
The PR to update the official React Navigation docs hasn't been merged yet. Here's the [deploy preview](https://deploy-preview-193--react-navigation.netlify.com/docs/en/redux-integration.html).

**Comment by mrikirill** [comment]
Thanks @Ashoat 
Would you recommend to update right now or wait till the official React Navigation doc will updated ?

**Comment by mrikirill** [comment]
Ok thanks @Ashoat I will update my app

**Comment by ViktorAksionov** [comment]
Example above doesn't work, also this
```
Some folks like to have their navigation state stored in the same place as the rest of their application state. Think twice before you consider doing this, there is an incredibly good chance that you do not need to do this!. Storing your React Navigation state in your own Redux store is likely to give you a very difficult time if you don't know what you're doing, and support for Redux integration is being depreacted.
```
is too bold statement. If you did not develop large app where you need to have strict control on each user move then without redux integration it will be easier just to abandon react-navigation and to move to other libs. Please don't be selfish person!

**Comment by YounetBet** [comment]
I want to know what to do with the android return key?

**Comment by Ashoat** [comment]
@YounetBet, I have no idea what you're talking about.

@ViktorAksionov, you're really calling open source volunteers selfish? Anyways, I didn't write that chunk. You can provide feedback on it [here](https://github.com/react-navigation/react-navigation/issues/4490). For reference, the "deprecated" chunk is going away.

**Comment by proutek** [comment]
Updated documentation states:
"4. A middleware is needed so that any events that mutate the navigation state properly trigger the event listeners. **To properly trigger the event listeners with the initial state, a call to initializeListeners is also necessary.**"

I have updated my project according to documentation but in example there is no explicit call to initializeListeners. I assume it is done in reduxifyNavigator now.

I'm asking because I have now problems with didFocus event which is not called.
